NicolaeChristof/Descend-Into-Darkness
# Feature first enemy

Added two new enemy types. The first is a skeleton that kills the player on sight! The green light in front of the skeleton indicates the death zone. The second enemy type is the skeleton archer which turns towards the player when they are in their cone of vision and shoots the player with projectiles. I also blocked out the mine level, you can find it in the maps folder.

How to test:
- get shot, make sure it really hurts. eventually you will die.
- run in front of the skeleton with the green light. It should murder you as soon as you get into it's light

Known Bugs:
- no known bugs (happy hunting)
